### ISSUE TITLE
Change documentation theme / revamp whole project identity?

### ISSUE THREAD

**Issue by astrojuanlu** [poliastro/poliastro#1474]
The sphinx_rtd_theme has served us well for many years (eight, to be exact) but there are nicer themes out there these days. In particular, these themes have a nice 3 column layout, some modern tweaks, and an eye on accessibility:

- Pradyun's Furo https://pradyunsg.me/furo/
- pydata-sphinx-theme https://pydata-sphinx-theme.readthedocs.io/en/latest/
- sphinx-book-theme https://github.com/executablebooks/sphinx-book-theme/

I don't quite like the "Material" themes (used by FastAPI and others) so I would personally ignore them.

There are several things I quite like about the current theme, like

- the slightly softer contrast
- the dark background of left sidebar
- the serif fonts for headings

In summary, there are a few ways to tackle this:

1. Don't do anything, and keep the current theme. Pros: it works and it isn't terrible. Cons: lots of wasted horizontal space, not very well maintained https://github.com/readthedocs/sphinx_rtd_theme/issues/295
2. Pick one of the themes above (or another one, with the condition that is well maintained) with minimal customization (our logo and not much else)
3. Customize one of the themes above more deeply (color, typography)
4. Commission a redesign of the documentation, website, and possibly a modernized version of the logo

If I'm opening this issue is obviously because I'd love to move past (1). Basically, I'm hesitating whether to pick (2), and then decide that we actually want (3) or even (4).

What do you think folks?

**Comment by airpods69** [comment]
Pradyun's Furo seems like a nice theme to me
Though I'd prefer to keep it just a bit darker

Doing (1) also works but then wasted space makes it a bit weird to read sometimes, so I will suggest to change it.

if you plan on a redesign and stuff then maybe i should be able to help with my really basic html css skills.

**Comment by astrojuanlu** [comment]
> if you plan on a redesign and stuff then maybe i should be able to help with my really basic html css skills.

Thanks a lot for offering your help! But for a redesign I would probably try to find someone that has at least more-than-really-basic HTML & CSS skills 😄 [The project has a bit of money](https://opencollective.com/poliastro), so maybe we could afford such a thing.

**Comment by jorgepiloto** [comment]
Honestly, the `pydata-sphinx-theme` is becoming the default in almos every package. It is well maintained, integrates with modern Sphinx extensions and is highly customizable.

**Comment by astrojuanlu** [comment]
Yep. It would be cool if poliastro could have something a bit more artistic, but for now, any of the themes mentioned in the first comment + a bit of color customization will be an improvement over the current one. It's just too outdated unfortunately.
